act-rules-format.md

```diff
@@ -206,6 +213,71 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Mapping Secondary Requirements ####
+An accessibility requirement is mapped as a secondary requirement in a rule when both of the following are true: 
```

Comment:
I overall like how these are setup, the only thing that feels slightly weird to me is that a conformance requirement technically meets the definition of a secondary requirement as well. Should we add some verbiage to the effect of using the stricter of the two? e.g., if you meet the conformance requirement tests you have to use that one.